### DISCUSSION TITLE
Pac1 does not match the output read from the generation counter

### DISCUSSION THREAD

**Discussion by shadowbyte68** [SBFspot/SBFspot#627] [General]
What is Pac1 for a value? How is this calculated?
Pac1 does not match the output read from the generation counter.
I have a problem with PVoutput because the export/import is calculated from generation and consumption. The consumption comes from the energy meter and is sent separately to PVoutput (v4).
Since Pac1 does not match the 5min value of the generation counter, the import/export of PVoutput is also incorrectly calculated

**Comment by sbf-** [comment]
Pac1 is retrieved from inverter. In some cases it is calculated with Vac1*Iac1 (values coming from inverter) - This is controlled by config `CalculateMissingSpotValues=1`
This is visible in verbose logging (-v5)
